doc/api/Miscellaneous/MultiThreading.md

```diff
@@ -216,52 +210,62 @@ have already been played on the main thread.
 
 ## How to rely on "multithread" mode
 
-### Step 1: Obtaining the Worker file and the WebAssembly file
-
-The RxPlayer will need to be able to obtain two files to be able to run in multithread
-mode:
+### Step 1: Obtaining the Worker file
 
-- `worker.js`: The "worker" file, which will run in a WebWorker concurrently to your
-  application.
+The RxPlayer will need to be able to obtain an external file to be able to run in
+multithread mode, `worker.js`, which is the code running in a WebWorker concurrently to
+your application.
 
-- `mpd-parser.wasm`: The DASH WebAssembly parser, today the only supported transport's
-  parser in a "multithread" scenario (Microsoft Smooth streaming, local contents and
-  Metaplaylist are not yet supported).
+You can find it at any of the following places:
 
-You can find them at any of the following places:
-
-- The easiest way is to just import in your application the "embedded" versions of each of
-  them, exported through the `"rx-player/experimental/features/embeds"` path:
+- The easiest way is to just import in your application its "embedded" version, exported
+  through the `"rx-player/experimental/features/embeds"` path:
 
   ```js
-  import {
-    EMBEDDED_WORKER, // "embedded" version of the `worker.js` file
-    EMBEDDED_DASH_WASM, // "embedded" version of the `mpd-parser.wasm` file
-  } from "rx-player/experimental/features/embeds";
+  import { EMBEDDED_WORKER } from "rx-player/experimental/features/embeds";
   ```
 
-  This allows to bypass the need to store and serve separately those two files. Note
-  however that including those "embeds" in your application may sensibly increase its
-  size.
+  This allows to bypass the need to store and serve separately those two files.
```

Comment:
> Those two files.

There is now only one thanks to that PR :)

Reply:
Updated!